plugins/domains/d.pr.js

```diff
@@ -4,14 +4,11 @@ module.exports = {
         /^https?:\/\/(\w+\.)?d\.pr\/(?:i\/)?([a-zA-Z0-9]+)/i
```

Comment:
can we please add `(?:i|v)` for videos to regex explicitly. Also, let's account for optional `free` as well.